ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -288,6 +289,26 @@ void OwInterface::managePanTilt (const string& opname,
 
   int id = Running.at (opname);
 
+  //if position is over 360 we want to bring it back within the
+  //-360 to 360 range to check if goal position has been reached.
+  if(fabs(current) > 360+DegreeTolerance){
```

Comment:
I don't understand the reason for adding **DegreeTolerance** at this line, can you elaborate?

Reply:
That shouldn't be there, I think that I was trying to test something and forgot to take it out. Will fix that.